wikidatarefisland/run.py

```diff
@@ -40,14 +44,23 @@ def main(argv, filepath):
         storage.store(args.output_path, ext_ids)
         return
 
+    if 'extract_items' == args.step:
+        try:
+            whitelisted_ext_ids = storage.get(args.side_service_input_path)
+        except FileNotFoundError:
```

Comment:
I don't think we need this here, the dependencies on disk should be expressed in the makefile. If we keep it here it would be harder to replace SS1 with a different implementation.

Reply:
cool! Done